app/lib/three_scale/search.rb

```diff
@@ -184,8 +184,13 @@ def table_and_column(column)
 
   module Helpers
 
+    # Maximum page size for search results
```

Comment:
```suggestion
    # Default maximum page size for search results
```
Otherwise it may be read as absolute maximum.